tensorrt_llm/_torch/models/checkpoints/mx/checkpoint_loader.py

```
@@ -0,0 +1,374 @@
+                "modelexpress_client/python). Falling back to disk loading."
+            )
+            return self._fallback_to_disk(checkpoint_dir, mapping, **kwargs)
+
+        try:
+            mx_loader = MxLiveWeightLoader(mx_server=self._mx_server_url)
+            fallback_weights = mx_loader.load_weights(
+                checkpoint_dir,
+                mapping=mapping,
+                model=model,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)